providers/summary/scancode.js

```diff
@@ -84,10 +91,26 @@ class ScanCodeSummarizer {
     if (!licenses) return null
     const list = setToArray(licenses)
     if (!list) return null
-    return list
-      .map(normalizeSpdx)
-      .filter(x => x)
-      .join(' AND ')
+    if (list.length === 1) return list[0]
+    return list.map(x => `(${x})`).join(' AND ')
+  }
+
+  _createExpression(rule, licenseKey) {
+    const license_expression = get(rule, 'license_expression')
+    if (!license_expression) return normalizeSpdx(licenseKey)
```

Comment:
Probably should consider the `rule_relevance` with some feedback from @pombredanne as to how to interpret. In the sample data it seems like two rules matched but one scored only 25 while the other 100. Not sure where the cut off should be.

Reply:
good point. totally agree - I can code up a cutoff at 50 and we can pick a number when we merge